flamby/benchmarks/fed_benchmark.py

```diff
@@ -211,20 +214,29 @@ def main(args_cli):
             df.drop(index_of_interest, inplace=True)
         m = copy.deepcopy(global_init)
         set_seed(args_cli.seed)
-        m = train_single_centric(
-            m,
-            train_pooled,
-            use_gpu,
-            "Pooled",
-            pooled_hyperparameters["optimizer_class"],
-            pooled_hyperparameters["learning_rate"],
-            BaselineLoss,
-            NUM_EPOCHS_POOLED,
-            dp_target_epsilon=pooled_hyperparameters["dp_target_epsilon"],
-            dp_target_delta=pooled_hyperparameters["dp_target_delta"],
-            dp_max_grad_norm=pooled_hyperparameters["dp_max_grad_norm"],
-            seed=args_cli.seed,
-        )
+        try:
+            m = train_single_centric(
+                m,
+                train_pooled,
+                use_gpu,
+                "Pooled",
+                pooled_hyperparameters["optimizer_class"],
+                pooled_hyperparameters["learning_rate"],
+                BaselineLoss,
+                NUM_EPOCHS_POOLED,
+                dp_target_epsilon=pooled_hyperparameters["dp_target_epsilon"],
+                dp_target_delta=pooled_hyperparameters["dp_target_delta"],
+                dp_max_grad_norm=pooled_hyperparameters["dp_max_grad_norm"],
+                seed=args_cli.seed,
+            )
+        except RuntimeError:
+            traceback.print_exc()
+            raise RuntimeError(
+                "The previous error might be linked to the use of phikon features. If"
+                " you want to use them make sure to have the parameter"
+                " --use-ssl-features enabled as well as the correct path in the"
+                " dataset_location.yaml file."
+            )
```

Comment:
I don't understand this I am in favor of removing it

Reply:
if a user did not use the `--use-ssl-features` and the training encounters a runtime error this will warn the user that the reason of the error may be to add this parameter. We can remove it if you want

Reply:
You are catching too much unrelated errors